docs/onboarding/Configuration-Properties.md

```diff
@@ -134,6 +134,23 @@ The Onboarding Server uses the following public configuration properties:
 | `enrollment-server-onboarding.presence-check.iproov.zenid.restClientConfig.proxyUsername`         |                           | Proxy username to be used when calling iProov REST service.                   |
 | `enrollment-server-onboarding.presence-check.iproov.zenid.restClientConfig.proxyPassword`         |                           | Proxy password to be used when calling iProov REST service.                   |
 
+
+## Innovatrics Configuration
+
+| Property                                                                                | Default                   | Note                                                                           |
+|-----------------------------------------------------------------------------------------|---------------------------|--------------------------------------------------------------------------------|
+| `enrollment-server-onboarding.innovatrics.serviceBaseUrl`                               |                           | Base REST service URL for Innovatrics.                                         |
```

Comment:
👀 Both Innovatrics providers' configuration is united, no discriminator for `presence-check` or `document-verifier`, just the `innovatrics` key.